demo/nodejs-domino/custom.ts

```diff
@@ -0,0 +1,53 @@
+import domino from 'domino'
+import './another'
+import {
+  trustedTypes,
+  TrustedTypeConfig,
+  TrustedTypesEnforcer,
+} from 'trusted-types'
+
+function acceptOnlyTrustedHTML(html: TrustedHTML) {
+  if (!trustedTypes.isHTML(html)) console.log('untrusted html', html)
+  else console.log('trusted html', html)
+}
+
+const policy = trustedTypes.createPolicy('app', { createHTML: (s) => s })
+acceptOnlyTrustedHTML('str' as any)
+acceptOnlyTrustedHTML(policy.createHTML('safe html'))
+
+const win = domino.createWindow()
+const doc = win.document
+
+const html = doc.createElement('html')
+html.appendChild(doc.createElement('div'))
+html.appendChild(doc.createElement('div'))
+html.appendChild(doc.createElement('div'))
+
+console.log(html.innerHTML)
+
+// no fail
+html.children[0].innerHTML = 'string'
+console.log(html.innerHTML)
+
+const config = new TrustedTypeConfig(false, true, ['foo'], false)
+const enforcer = new TrustedTypesEnforcer(config, win)
+enforcer.install()
+const fooPolicy = trustedTypes.createPolicy('foo', { createHTML: (s) => s })
+try {
+  html.children[0].innerHTML = 'string' // should fail
+  throw new Error('Unexpected code path')
```

Comment:
This throw makes it confusing, as you don't know in the exception handler below if the `innerHTML` actually succeeded (this is what was breaking the default policy code we talked about).